react/index.js

```diff
@@ -248,21 +260,21 @@ class ProductSummary extends Component {
     } = this.props
 
     const buyButtonClasses = classNames(
-      'vtex-product-summary__buy-button-container pv3 w-100',
+      `${productSummary.buyButtonContainer} pv3 w-100`,
       {
         'dn': displayMode === 'small' || displayMode === 'inline',
         'dn db-ns': displayMode === 'normal',
       }
     )
-    
-    const showBuyButton =  !showButtonOnHover || mobile || this.state.isHovering
+
+    const showBuyButton = !showButtonOnHover || mobile || this.state.isHovering
     const quantity = path(['sku', 'seller', 'commertialOffer', 'AvailableQuantity'], product) || 0
     const isAvailable = (quantity > 0)
 
     return (
       !hideBuyButton && (
-        <div className={ buyButtonClasses }>
-          <div className={`vtex-product-summary__buy-button center mw-100 ${ !showBuyButton && 'is-hidden' }`}>
+        <div className={buyButtonClasses}>
+          <div className={`${productSummary.buyButton} center mw-100 ${!showBuyButton && 'is-hidden'}`}>
```

Comment:
if `!showBuyButton` is `false` the output of 
```js
`${!showBuyButton && 'is-hidden'}`
```
is `'false'` instead of just an empty string

Reply:
So it should be something like 
```js
`${!showBuyButton ? 'is-hidden' : ''}`
```